openhands/runtime/client/runtime.py

```diff
@@ -557,9 +557,11 @@ def list_files(self, path: str | None = None) -> list[str]:
         except Exception as e:
             raise RuntimeError(f'List files operation failed: {str(e)}')
 
-    def zip_files_in_sandbox(self) -> bytes:
-        """Zips the files in the sandbox and returns the bytes for streaming."""
-        sandbox_dir = os.getcwd() + self.config.workspace_mount_path_in_sandbox
+    def copy_from(self, src: str | None = None) -> bytes:
```

Comment:
@xingyaoww there's another runtime where we need to implement this, right?

Reply:
yes - i think we at least need to do this for `EventStream` and `RemoteRuntime`